sentry-rails/lib/sentry/rails/background_worker.rb

```diff
@@ -1,6 +1,6 @@
 module Sentry
   class BackgroundWorker
-    def perform(&block)
+    def _perform(&block)
       @executor.post do
```

Comment:
since this is already inside the `@executor.post` in `perform`, I think we don't need it here anymore?

Reply:
@sl0thentr0py great catch, thanks 👍